bin/jib.sh

```diff
@@ -128,6 +128,19 @@ install_jib() {
             exit 1
         fi
     fi
+    # Want to check the filetype using file, to see if we got served a HTML error page.
+    # In order to support solaris, we don't use any fancy flags to file.
+    # file will prepend the filename, so we need to remove it before testing the output.
+    PREFIX="${installed_jib_script}.gz: "
+    FILEOUTPUT=`file ${installed_jib_script}.gz`
+    # ${X:${#Y}} gives X without the first ${#Y} characters, and ${#Y} is length of Y.
+    FILEOUTPUT=${FILEOUTPUT:${#PREFIX}}
```

Comment:
I think this is a bashism. While we do have /bin/bash in the shebang, this does not really matter since we have no executable bit on the script, so users will have to explicitly call it with a shell. So many users are likely to do `sh bin/jib.sh`, and then this will break.

My recommendation would be to use standard tools like cut or grep instead, something like:
```
file $installedscript.gz | grep "gzip compressed data" -q
if test $? -neq 0; then 
...
```

Reply:
(Warning: untested code)

I also would like to clarify the use of bash. For configure script, and scripts started by the makefiles, we rely on and enforce bash, so in those cases bashisms are accepted (and almost encouraged, at least good ones). But in "user facing" scripts like this, or the top level `configure` script, we want to make minimal assumptions about the shell used, since the shebang path means nothing in our case.

Reply:
@magicus I've changed to something inspired by your comment and tested on linux, solaris and macos. Do you think the current version would be ok?